cmd/vic-machine/converter/converter.go

```diff
@@ -328,6 +329,19 @@ func convertIP(src reflect.Value, prefix string, tags reflect.StructTag, dest ma
 	return nil
 }
 
+func convertTrustLevel(src reflect.Value, prefix string, tags reflect.StructTag, dest map[string][]string) error {
+	log.Debugf("convertTrustLevel: prefix: %s, src: %s", prefix, src.String())
+	trustLevel, ok := src.Interface().(common.TrustLevel)
+	if !ok {
+		panic(fmt.Sprintf(src.Type().String() + " is not TrustLevel"))
```

Comment:
right, this panic is just for someone put this type's converter to convert other type's object.
Any expected error should be returned